packages/app/pages/checkout/[network]/[address].tsx

```diff
@@ -171,6 +207,10 @@ const CheckoutPage = ({
             console.log("successfully bought", id, "!");
             setLoading(false);
             setSupply(supply - 1);
+            if (url) {
+              // If valid redirection URL is provided, redirect on successful purchase.
+              window.location.assign(url);
+            }
```

Comment:
this is good for a v1. I think for v2, we'll want to delay this on a timeout and display a success toast

Reply:
agreed. added a TODO